web/styles/settings.css

```diff
@@ -672,6 +674,15 @@ input[type="checkbox"] {
     }
 }
 
+.required-symbol {
+    color: hsl(0deg 66% 60%);
+}
+
+.field {
+    margin-top: 5px;
+    max-width: fit-content;
+}
```

Comment:
This sort of refactoring to create an extremely generic `field` CSS rule is very dangerous. The following grep shows how this will modify how bot avatar rows, the change password form, etc. are rendered, which seems not the intent.
``` console
$ git grep class= | grep [^-_]field[^-_]
templates/corporate/for/business.html:                    <li><div class="list-content">Use Zulip as your employee directory. Sync names, <a href="/help/roles-and-permissions">roles</a>, avatars, and <a href="/help/custom-profile-fields">custom profile fields</a> from LDAP or SAML.</div></li>
templates/corporate/support/support_request.html:            <fieldset class="support-request">
templates/zerver/register.html:            <fieldset class="org-registration">
templates/zerver/register.html:            <fieldset class="user-registration">
web/templates/dialog_change_password.hbs:    <div class="field password-div">
web/templates/dialog_change_password.hbs:    <div class="field password-div">
web/templates/settings/admin_profile_field_list.hbs:        <button class="button rounded small btn-warning open-edit-form-modal tippy-zulip-delayed-tooltip" data-tippy-content="{{t 'Edit custom profile field' }}" data-profile-field-id="{{id}}">
web/templates/settings/admin_profile_field_list.hbs:        <button class="button rounded small delete btn-danger tippy-zulip-delayed-tooltip" data-tippy-content="{{t 'Delete custom profile field' }}" data-profile-field-id="{{id}}">
web/templates/settings/bot_avatar_row.hbs:            <div class="field">{{t "Bot type" }}</div>
web/templates/settings/bot_avatar_row.hbs:            <div class="field">{{t "Bot email" }}</div>
web/templates/settings/bot_avatar_row.hbs:            <span class="field">{{t "API key" }}</span>
web/templates/settings/custom_user_profile_field.hbs:<div class="custom_user_field" name="{{ field.name }}" data-field-id="{{ field.id }}">
web/templates/settings/custom_user_profile_field.hbs:        <label class="inline-block {{#if field.required}}required-field{{/if}}" for="{{ field.name }}" class="title">{{ field.name }}</label>
web/templates/settings/custom_user_profile_field.hbs:    <div class="field_hint">{{ field.hint }}</div>
web/templates/settings/custom_user_profile_field.hbs:    <div class="field {{#if is_empty_required_field}}empty-required-field{{/if}}">
web/templates/settings/custom_user_profile_field.hbs:        <input class="custom_user_field_value datepicker {{#if for_manage_user_modal}}modal_text_input{{else}}settings_text_input{{/if}}" data-field-id="{{ field.id }}" type="text"
web/templates/settings/profile_field_settings_admin.hbs:        <button class="button rounded sea-green" id="add-custom-profile-field-btn">{{t "Add a new profile field" }}</button>
web/templates/settings/profile_field_settings_admin.hbs:            <tbody id="admin_profile_fields_table" class="required-text" data-empty="{{t 'No custom profile field configured for this organization.' }}"></tbody>
web/templates/settings_overlay.hbs:                        <div class="text">{{t "Custom profile fields" }}</div>
````

Never do this sort of refactor that could impact random other parts of the app inside a giant commit adding a feature; it's a very very easy way to break unexpected things, and takes a lot of attention from a reviewer to notice you've done it. You should always do it in its own commit, which draws attention to the change, both for yourself and for reviewers, and forces you to think about writing a commit message for how you can provide it's correct ... or whether it's in fact not.

Reply:
I'm adding three little prep commits removing the other users of `.field` as a class name, and also renamed `.field` here as well.

@karlstolley FYI -- these commits touch very lightly on the settings CSS changes you're doing, so you might get some logical or otherwise merge commits with your `dropdown-title` work.